samcli/lib/build/app_builder.py

```diff
@@ -110,6 +110,7 @@ def __init__(
         container_env_var: Optional[Dict] = None,
         container_env_var_file: Optional[str] = None,
         build_images: Optional[Dict] = None,
+        excluded_resources: Optional[Tuple[str, ...]] = None,
```

Comment:
I suppose we don't need to add the argument?

Reply:
Right, since now the `excluded_resources` isn't included in the app builder